main.tf

```diff
@@ -102,15 +107,16 @@ locals {
     container_name   = coalesce(var.alb_container_name, module.this.id)
     container_port   = var.container_port
     elb_name         = null
-    target_group_arn = module.alb_ingress.target_group_arn
+    target_group_arn = local.alb_enabled ? module.alb_ingress.target_group_arn : ""
   }
   nlb = {
     container_name   = coalesce(var.nlb_container_name, module.this.id)
     container_port   = var.nlb_container_port
     elb_name         = null
     target_group_arn = var.nlb_ingress_target_group_arn
   }
-  load_balancers = var.nlb_ingress_target_group_arn != "" ? [local.alb, local.nlb] : [local.alb]
+  alb_load_balancers = local.alb_enabled ? [local.alb] : []
+  load_balancers     = var.nlb_ingress_target_group_arn != "" ? concat(local.alb_load_balancers, [local.nlb]) : local.alb_load_balancers
```

Comment:
The more I look at this, I realize that were being inconsistent here. We're using `var.nlb_ingress_target_grouo_arn` to determine if we add the nlb to the list of load balancers. We already have a similar variable for the alb. Why don't we use that same logic for alb and then we don't have to add an additional variable.

Reply:
Easy enough update, I drop the `alb_enabled` variable

Reply:
Looks like this module doesn't support NLB creation, but does support ALB creation. I think that bit of inconsistency prompted the additional variable fwiw

Reply:
Exploring the TF that it looks like an alb also isn't created within this module, looking at bit at the collection of module I might have missed some of the configuration of submodules. I might have been able  to set `alb_ingress_enable_default_target_group` to `false` to achieve simpler behavior

Reply:
@gausnes is this change no longer needed ?